File: modules/custom-button-fabric/ios/CustomButtonContainerManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(CustomButtonContainerManager, RCTViewManager)

// Export properties
RCT_EXPORT_VIEW_PROPERTY(buttonTitle1, NSString)
RCT_EXPORT_VIEW_PROPERTY(buttonTitle2, NSString)
RCT_EXPORT_VIEW_PROPERTY(buttonColor1, NSString)
RCT_EXPORT_VIEW_PROPERTY(buttonColor2, NSString)
RCT_EXPORT_VIEW_PROPERTY(onButtonPress, RCTBubblingEventBlock)

@end

File: modules/custom-button-fabric/ios/CustomButtonContainerManager.swift
import React

#if !RCT_NEW_ARCH_ENABLED

@objc(CustomButtonContainerManager)
public class CustomButtonContainerManager: RCTViewManager {
  
  override public func view() -> UIView {
    return CustomButtonContainerLegacyView()
  }
  
  override public static func requiresMainQueueSetup() -> Bool {
    return false
  }
}

#endif
